server/app.js

```diff
@@ -0,0 +1,32 @@
+const config = require('./utils/config')
```

Comment:
Although, it is a matter of preference the `eslint` best practice is to destructure your import like this 
```javascript
const { MONGODB_URI } = require('./utils/config')'
```
This is the pattern we are already following in the frontend, let's be consistent, please. So, I am requesting the same pattern for all instances of imports from the `./utils.`